packages/kamino-sdk/src/Kamino.ts

```diff
@@ -250,6 +264,73 @@ export class Kamino {
     }
   };
 
+  getRebalanceMethods = (): RebalanceMethod[] => {
+    return [ManualRebalanceMethod, PricePercentageRebalanceMethod];
+  };
+
+  getDefaultRebalanceMethod = (): RebalanceMethod => ManualRebalanceMethod;
+
+  getDefaultRebalanceFields = async (
+    dex: Dex,
+    poolTokenA: PublicKey,
+    poolTokenB: PublicKey,
+    rebalanceMethod: string
+  ): Promise<RebalanceFieldInfo[]> => {
+    if (dex == 'ORCA') {
+      let pools = await this.getOrcaPoolsForTokens(poolTokenA, poolTokenB);
+      if (pools.length == 0) {
+        throw new Error(`No pool found for ${poolTokenA.toString()} and ${poolTokenB.toString()}`);
+      }
+
+      let price = pools[0].price;
+      if (rebalanceMethod == PricePercentageRebalanceMethod.label) {
+        let lowerPrice = 0.95 * price;
+        let upperPrice = 1.05 * price;
+
+        let fieldInfos = getPricePercentageRebalanceFieldInfos(
+          DefaultLowerPercentageBPS,
+          DefaultUpperPercentageBPS
+        ).concat(getManualRebalanceFieldInfos(lowerPrice, upperPrice));
```

Comment:
I think you need to add an `enabled` or `disabled` value here because in the PricePercentage case the price cannot be modified. 

also we need a `recalculatePriceFieldsOnMethodUpdate()`

so when they change the %, then the new prices appear. logic should live in the SDK

Reply:
I am not sure I understand what you mean; what I want to do here is to return a price range for the intial position, and set the rebalance percentages